3p/integration.js

```diff
@@ -105,6 +106,23 @@ const AMP_EMBED_ALLOWED = {
   zergnet: true,
 };
 
+// used for extracting fakead3p from production code.
+const IS_DEV = true;
+
+if (IS_DEV) {
+  let toRegister = false;
+  const parseData = parseFragment(location.hash);
```

Comment:
As discussed offline, we will fragment parsing logic out of the blog, as it also appeared in `draw3p` function. So that we don't need this magic `IS_DEV` here, and can directly use mode.localDev.